Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:latest
+
+RUN apt-get update && apt-get install -y pdftk texlive-latex-base texlive-latex-extra texlive-fonts-recommended texlive-fonts-extra
```

Comment:
As we have an essentially "fresh" linux install with nothing more than some basic python tools, we install all system packages our program needs to operate.